docs/networking/networking.md

```diff
@@ -76,6 +83,14 @@ To disable the embedded LB, configure all servers in the cluster with the `--dis
 
 This is necessary if you wish to run a different LB, such as MetalLB.
 
+## External Cloud Controller Manager
+
+K3s ships with a stub Cloud Controller Manager (CCM) that does not actually implement any functionality. In order to use an external CCM, you must start K3s with the `--disable-cloud-controller` flag. 
+
+:::note K3s v1.22 and older
+In order to deploy an external CCM with K3s v1.22 and older, you must also start K3s with the `--kubelet-arg="cloud-provider=external"` flag.
```

Comment:
I don't recall this being necessary, did we not previously set that flag when the stub CCM is disabled?

Reply:
This seems to have been necessary way back in v1.17 and older, I mangled that with the fact that in v1.23+ the `cloud-provider` arg is no longer a thing (its always external for kubelet). Removed!